android/src/facebook/TiFacebookModule.java

```diff
@@ -115,6 +115,20 @@ public static void onAppCreate(TiApplication app)
 	public static TiFacebookModule getFacebookModule() {
 		return module;
 	}
+    
+    @Override
+    public void onResume(Activity activity) {
+        super.onResume(activity);
+        Log.d(TAG, "Calling activateApp");
+        AppEventsLogger.activateApp(activity);
```

Comment:
@ashcoding , could you move this issue in some direction, because I got a new bug that I also need merged :) It's already been almost a month that we released the app with this fix in Google Play, everyone seems happy :)